### ISSUE TITLE
Parser config

### ISSUE THREAD

**Issue by ntotten** [dangmai/prettier-plugin-apex#106]
I think that the way the config is set for anon apex isn't correct to the prettier way of doing things. This should really be a different parser. So you would register two parsers with prettier for example: `apex` and `anon-apex`. Then you can set the config like this to specify which to use in the `.prettierrc` file:

```
{
  "semi": false,
  "overrides": [
    {
      "files": "*.apex",
      "options": {
        "parser": "anon-apex"
      }
    },
    {
      "files": "*.cls",
      "options": {
        "parser": "apex"
      }
    }
  ]
}
```

I am just thinking of how we would set the parser correctly once the VS Code Prettier supports plugins. We need some way to set it globally and not for each run of the formatter.

**Comment by dangmai** [comment]
Thanks for the bug ticket. I think you're right, now that I think about it more it makes sense that anonymous Apex should be treated as a different parser than named Apex. I'll make a fix for this soon.

**Comment by dangmai** [comment]
I have fixed this in `master`, it'll be included in `1.0.0`. By the way, do you recommend using the `.apex` extension for anonymous Apex? By default, the plugin picks up `.cls` and `.trigger` but maybe `.apex` should be included as well.
